docs/src/languages/php/_index.md

```diff
@@ -106,133 +156,9 @@ In other words, it's equivalent to the following `composer.json` file:
-
-## Build flavor
-
-PHP images use the `composer` build flavor by default,
-which runs `composer --no-ansi --no-interaction install --no-progress --prefer-dist --optimize-autoloader` if a `composer.json` file is detected.
-
-Note that by default, all PHP containers include the latest Composer 1.x release.
-If you wish to use Composer 2.x, add it as a `dependency`:
-
-```yaml
-dependencies:
-    php:
-        composer/composer: '^2'
-```
-
-## OPcache preloading
-
-From PHP 7.4, you can use OPcache preloading,
-which allows you to load selected files into shared memory when PHP-FPM starts.
-That means functions and classes in those files are always available and don't need to be autoloaded,
-at the cost of any changes to those files requiring a PHP-FPM restart.
-Since PHP-FPM restarts on each new deploy, this feature is a major win on Platform.sh and we recommend using it aggressively.
-
-To enable preloading, add a `php.ini` value that specifies a preload script.
-Any [`php.ini` mechanism](/languages/php/ini.md) works,
-but using a variable in `.platform.app.yaml` is the recommended approach:
-
-```yaml
-variables:
-    php:
-        opcache.preload: 'preload.php'
-```
-
-The `opcache.preload` value is evaluated as a file path relative your [app configuration](../../create-apps/_index.md).
-It may be any PHP script that calls `opcache_compile_file()`.
-
-The following example preloads all `.php` files anywhere in the `vendor` directory:
-
-```php
-<?php
-$directory = new RecursiveDirectoryIterator(getenv('PLATFORM_APP_DIR') . '/vendor');
-$iterator = new RecursiveIteratorIterator($directory);
-$regex = new RegexIterator($iterator, '/^.+\.php$/i', RecursiveRegexIterator::GET_MATCH);
-
-foreach ($regex as $key => $file) {
-    // This is the important part!
-    opcache_compile_file($file[0]);
-}
-```
-
-{{< note >}}
-
-Preloading all `.php` files may not be optimal for your application and may even introduce errors.
-Your application framework may provide recommendations or a pre-made preload script to use instead.
-You have to determine the optimal preloading strategy for your situation.
-
-{{< /note >}}
-
-#### Preloading and dependencies
-
-Your preload script runs each time PHP-FPM restarts, including during your build.
-This means it runs before your dependencies have been installed (such as with Composer).
-
-If your preload script uses `require` for dependencies, it fails during the build
-because the dependencies aren't yet present.
-
-To resolve this, you have two options:
-
-* Have your script `include` dependencies instead of `require`
-  and fail gracefully if the dependencies aren't there.
-* Enable preloading with a variable that [isn't available during the build](../../development/variables/set-variables.md#variable-options).
-  Then preloading happens only on deploy.
-
-## FFI
-
-PHP 7.4 introduced support for Foreign Function Interfaces (FFI),
```

Comment:
:lady_beetle: I didn't see below that it was only from 7.4+. Might be worth adding, as noted below.